### ISSUE TITLE
Glyphs: Draw Latin Plus

### ISSUE THREAD

**Issue by sberlow** [TypeNetwork/Amstelvar#43]
- [x] Roman
- [x] Italic
- [ ] Chinese

Latin Plus:
`0x0100 Ā LATIN CAPITAL LETTER A WITH MACRON`
`0x0101 ā LATIN SMALL LETTER A WITH MACRON`
`0x0102 Ă LATIN CAPITAL LETTER A WITH BREVE`
`0x0103 ă LATIN SMALL LETTER A WITH BREVE`
`0x0104 Ą LATIN CAPITAL LETTER A WITH OGONEK`
`0x0105 ą LATIN SMALL LETTER A WITH OGONEK`
`0x0106 Ć LATIN CAPITAL LETTER C WITH ACUTE`
`0x0107 ć LATIN SMALL LETTER C WITH ACUTE`
`0x0108 Ĉ LATIN CAPITAL LETTER C WITH CIRCUMFLEX`
`0x0109 ĉ LATIN SMALL LETTER C WITH CIRCUMFLEX`
`0x010A Ċ LATIN CAPITAL LETTER C WITH DOT ABOVE`
`0x010B ċ LATIN SMALL LETTER C WITH DOT ABOVE`
`0x010C Č LATIN CAPITAL LETTER C WITH CARON`
`0x010D č LATIN SMALL LETTER C WITH CARON`
`0x010E Ď LATIN CAPITAL LETTER D WITH CARON`
`0x010F ď LATIN SMALL LETTER D WITH CARON`
`0x0110 Đ LATIN CAPITAL LETTER D WITH STROKE`
`0x0111 đ LATIN SMALL LETTER D WITH STROKE`
`0x0112 Ē LATIN CAPITAL LETTER E WITH MACRON`
`0x0113 ē LATIN SMALL LETTER E WITH MACRON`
`0x0114 Ĕ LATIN CAPITAL LETTER E WITH BREVE`
`0x0115 ĕ LATIN SMALL LETTER E WITH BREVE`
`0x0116 Ė LATIN CAPITAL LETTER E WITH DOT ABOVE`
`0x0117 ė LATIN SMALL LETTER E WITH DOT ABOVE`
`0x0118 Ę LATIN CAPITAL LETTER E WITH OGONEK`
`0x0119 ę LATIN SMALL LETTER E WITH OGONEK`
`0x011A Ě LATIN CAPITAL LETTER E WITH CARON`
`0x011B ě LATIN SMALL LETTER E WITH CARON`
`0x011C Ĝ LATIN CAPITAL LETTER G WITH CIRCUMFLEX`
`0x011D ĝ LATIN SMALL LETTER G WITH CIRCUMFLEX`
`0x011E Ğ LATIN CAPITAL LETTER G WITH BREVE`
`0x011F ğ LATIN SMALL LETTER G WITH BREVE`
`0x0120 Ġ LATIN CAPITAL LETTER G WITH DOT ABOVE`
`0x0121 ġ LATIN SMALL LETTER G WITH DOT ABOVE`
`0x0122 Ģ LATIN CAPITAL LETTER G WITH CEDILLA`
`0x0123 ģ LATIN SMALL LETTER G WITH CEDILLA`
`0x0124 Ĥ LATIN CAPITAL LETTER H WITH CIRCUMFLEX`
`0x0125 ĥ LATIN SMALL LETTER H WITH CIRCUMFLEX`
`0x0126 Ħ LATIN CAPITAL LETTER H WITH STROKE`
`0x0127 ħ LATIN SMALL LETTER H WITH STROKE`
`0x0128 Ĩ LATIN CAPITAL LETTER I WITH TILDE`
`0x0129 ĩ LATIN SMALL LETTER I WITH TILDE`
`0x012A Ī LATIN CAPITAL LETTER I WITH MACRON`
`0x012B ī LATIN SMALL LETTER I WITH MACRON`
`0x012C Ĭ LATIN CAPITAL LETTER I WITH BREVE`
`0x012D ĭ LATIN SMALL LETTER I WITH BREVE`
`0x012E Į LATIN CAPITAL LETTER I WITH OGONEK`
`0x012F į LATIN SMALL LETTER I WITH OGONEK`
`0x0130 İ LATIN CAPITAL LETTER I WITH DOT ABOVE`
`0x0134 Ĵ LATIN CAPITAL LETTER J WITH CIRCUMFLEX`
`0x0135 ĵ LATIN SMALL LETTER J WITH CIRCUMFLEX`
`0x0136 Ķ LATIN CAPITAL LETTER K WITH CEDILLA`
`0x0137 ķ LATIN SMALL LETTER K WITH CEDILLA`
`0x0138 ĸ LATIN SMALL LETTER KRA`
`0x0139 Ĺ LATIN CAPITAL LETTER L WITH ACUTE`
`0x013A ĺ LATIN SMALL LETTER L WITH ACUTE`
`0x013B Ļ LATIN CAPITAL LETTER L WITH CEDILLA`
`0x013C ļ LATIN SMALL LETTER L WITH CEDILLA`
`0x013D Ľ LATIN CAPITAL LETTER L WITH CARON`
`0x013E ľ LATIN SMALL LETTER L WITH CARON`
`0x013F Ŀ LATIN CAPITAL LETTER L WITH MIDDLE DOT`
`0x0140 ŀ LATIN SMALL LETTER L WITH MIDDLE DOT`
`0x0141 Ł LATIN CAPITAL LETTER L WITH STROKE`
`0x0142 ł LATIN SMALL LETTER L WITH STROKE`
`0x0143 Ń LATIN CAPITAL LETTER N WITH ACUTE`
`0x0144 ń LATIN SMALL LETTER N WITH ACUTE`
`0x0145 Ņ LATIN CAPITAL LETTER N WITH CEDILLA`
`0x0146 ņ LATIN SMALL LETTER N WITH CEDILLA`
`0x0147 Ň LATIN CAPITAL LETTER N WITH CARON`
`0x0148 ň LATIN SMALL LETTER N WITH CARON`
`0x014A Ŋ LATIN CAPITAL LETTER ENG`
`0x014B ŋ LATIN SMALL LETTER ENG`
`0x014C Ō LATIN CAPITAL LETTER O WITH MACRON`
`0x014D ō LATIN SMALL LETTER O WITH MACRON`
`0x014E Ŏ LATIN CAPITAL LETTER O WITH BREVE`
`0x014F ŏ LATIN SMALL LETTER O WITH BREVE`
`0x0150 Ő LATIN CAPITAL LETTER O WITH DOUBLE ACUTE`
`0x0151 ő LATIN SMALL LETTER O WITH DOUBLE ACUTE`
`0x0154 Ŕ LATIN CAPITAL LETTER R WITH ACUTE`
`0x0155 ŕ LATIN SMALL LETTER R WITH ACUTE`
`0x0156 Ŗ LATIN CAPITAL LETTER R WITH CEDILLA`
`0x0157 ŗ LATIN SMALL LETTER R WITH CEDILLA`
`0x0158 Ř LATIN CAPITAL LETTER R WITH CARON`
`0x0159 ř LATIN SMALL LETTER R WITH CARON`
`0x015A Ś LATIN CAPITAL LETTER S WITH ACUTE`
`0x015B ś LATIN SMALL LETTER S WITH ACUTE`
`0x015C Ŝ LATIN CAPITAL LETTER S WITH CIRCUMFLEX`
`0x015D ŝ LATIN SMALL LETTER S WITH CIRCUMFLEX`
`0x015E Ş LATIN CAPITAL LETTER S WITH CEDILLA`
`0x015F ş LATIN SMALL LETTER S WITH CEDILLA`
`0x0160 Š LATIN CAPITAL LETTER S WITH CARON`
`0x0161 š LATIN SMALL LETTER S WITH CARON`
`0x0162 Ţ LATIN CAPITAL LETTER T WITH CEDILLA`
`0x0163 ţ LATIN SMALL LETTER T WITH CEDILLA`
`0x0164 Ť LATIN CAPITAL LETTER T WITH CARON`
`0x0165 ť LATIN SMALL LETTER T WITH CARON`
`0x0166 Ŧ LATIN CAPITAL LETTER T WITH STROKE`
`0x0167 ŧ LATIN SMALL LETTER T WITH STROKE`
`0x0168 Ũ LATIN CAPITAL LETTER U WITH TILDE`
`0x0169 ũ LATIN SMALL LETTER U WITH TILDE`
`0x016A Ū LATIN CAPITAL LETTER U WITH MACRON`
`0x016B ū LATIN SMALL LETTER U WITH MACRON`
`0x016C Ŭ LATIN CAPITAL LETTER U WITH BREVE`
`0x016D ŭ LATIN SMALL LETTER U WITH BREVE`
`0x016E Ů LATIN CAPITAL LETTER U WITH RING ABOVE`
`0x016F ů LATIN SMALL LETTER U WITH RING ABOVE`
`0x0170 Ű LATIN CAPITAL LETTER U WITH DOUBLE ACUTE`
`0x0171 ű LATIN SMALL LETTER U WITH DOUBLE ACUTE`
`0x0172 Ų LATIN CAPITAL LETTER U WITH OGONEK`
`0x0173 ų LATIN SMALL LETTER U WITH OGONEK`
`0x0174 Ŵ LATIN CAPITAL LETTER W WITH CIRCUMFLEX`
`0x0175 ŵ LATIN SMALL LETTER W WITH CIRCUMFLEX`
`0x0176 Ŷ LATIN CAPITAL LETTER Y WITH CIRCUMFLEX`
`0x0177 ŷ LATIN SMALL LETTER Y WITH CIRCUMFLEX`
`0x0178 Ÿ LATIN CAPITAL LETTER Y WITH DIAERESIS`
`0x0179 Ź LATIN CAPITAL LETTER Z WITH ACUTE`
`0x017A ź LATIN SMALL LETTER Z WITH ACUTE`
`0x017B Ż LATIN CAPITAL LETTER Z WITH DOT ABOVE`
`0x017C ż LATIN SMALL LETTER Z WITH DOT ABOVE`
`0x017D Ž LATIN CAPITAL LETTER Z WITH CARON`
`0x017E ž LATIN SMALL LETTER Z WITH CARON`
`0x018F Ə LATIN CAPITAL LETTER SCHWA`
`0x0192 ƒ LATIN SMALL LETTER F WITH HOOK`
`0x01A0 Ơ LATIN CAPITAL LETTER O WITH HORN`
`0x01A1 ơ LATIN SMALL LETTER O WITH HORN`
`0x01AF Ư LATIN CAPITAL LETTER U WITH HORN`
`0x01B0 ư LATIN SMALL LETTER U WITH HORN`
`0x01C4 Ǆ LATIN CAPITAL LETTER DZ WITH CARON`
`0x01C5 ǅ LATIN CAPITAL LETTER D WITH SMALL LETTER Z WITH CARON`
`0x01C6 ǆ LATIN SMALL LETTER DZ WITH CARON`
`0x01C7 Ǉ LATIN CAPITAL LETTER LJ`
`0x01C8 ǈ LATIN CAPITAL LETTER L WITH SMALL LETTER J`
`0x01C9 ǉ LATIN SMALL LETTER LJ`
`0x01CA Ǌ LATIN CAPITAL LETTER NJ`
`0x01CB ǋ LATIN CAPITAL LETTER N WITH SMALL LETTER J`
`0x01CC ǌ LATIN SMALL LETTER NJ`
`0x01E6 Ǧ LATIN CAPITAL LETTER G WITH CARON`
`0x01E7 ǧ LATIN SMALL LETTER G WITH CARON`
`0x01EA Ǫ LATIN CAPITAL LETTER O WITH OGONEK`
`0x01EB ǫ LATIN SMALL LETTER O WITH OGONEK`
`0x01FA Ǻ LATIN CAPITAL LETTER A WITH RING ABOVE AND ACUTE`
`0x01FB ǻ LATIN SMALL LETTER A WITH RING ABOVE AND ACUTE`
`0x01FC Ǽ LATIN CAPITAL LETTER AE WITH ACUTE`
`0x01FD ǽ LATIN SMALL LETTER AE WITH ACUTE`
`0x01FE Ǿ LATIN CAPITAL LETTER O WITH STROKE AND ACUTE`
`0x01FF ǿ LATIN SMALL LETTER O WITH STROKE AND ACUTE`
`0x0200 Ȁ LATIN CAPITAL LETTER A WITH DOUBLE GRAVE`
`0x0201 ȁ LATIN SMALL LETTER A WITH DOUBLE GRAVE`
`0x0202 Ȃ LATIN CAPITAL LETTER A WITH INVERTED BREVE`
`0x0203 ȃ LATIN SMALL LETTER A WITH INVERTED BREVE`
`0x0204 Ȅ LATIN CAPITAL LETTER E WITH DOUBLE GRAVE`
`0x0205 ȅ LATIN SMALL LETTER E WITH DOUBLE GRAVE`
`0x0206 Ȇ LATIN CAPITAL LETTER E WITH INVERTED BREVE`
`0x0207 ȇ LATIN SMALL LETTER E WITH INVERTED BREVE`
`0x0208 Ȉ LATIN CAPITAL LETTER I WITH DOUBLE GRAVE`
`0x0209 ȉ LATIN SMALL LETTER I WITH DOUBLE GRAVE`
`0x020A Ȋ LATIN CAPITAL LETTER I WITH INVERTED BREVE`
`0x020B ȋ LATIN SMALL LETTER I WITH INVERTED BREVE`
`0x020C Ȍ LATIN CAPITAL LETTER O WITH DOUBLE GRAVE`
`0x020D ȍ LATIN SMALL LETTER O WITH DOUBLE GRAVE`
`0x020E Ȏ LATIN CAPITAL LETTER O WITH INVERTED BREVE`
`0x020F ȏ LATIN SMALL LETTER O WITH INVERTED BREVE`
`0x0210 Ȑ LATIN CAPITAL LETTER R WITH DOUBLE GRAVE`
`0x0211 ȑ LATIN SMALL LETTER R WITH DOUBLE GRAVE`
`0x0212 Ȓ LATIN CAPITAL LETTER R WITH INVERTED BREVE`
`0x0213 ȓ LATIN SMALL LETTER R WITH INVERTED BREVE`
`0x0214 Ȕ LATIN CAPITAL LETTER U WITH DOUBLE GRAVE`
`0x0215 ȕ LATIN SMALL LETTER U WITH DOUBLE GRAVE`
`0x0216 Ȗ LATIN CAPITAL LETTER U WITH INVERTED BREVE`
`0x0217 ȗ LATIN SMALL LETTER U WITH INVERTED BREVE`
`0x0218 Ș LATIN CAPITAL LETTER S WITH COMMA BELOW`
`0x0219 ș LATIN SMALL LETTER S WITH COMMA BELOW`
`0x021A Ț LATIN CAPITAL LETTER T WITH COMMA BELOW`
`0x021B ț LATIN SMALL LETTER T WITH COMMA BELOW`
`0x022A Ȫ LATIN CAPITAL LETTER O WITH DIAERESIS AND MACRON`
`0x022B ȫ LATIN SMALL LETTER O WITH DIAERESIS AND MACRON`
`0x022C Ȭ LATIN CAPITAL LETTER O WITH TILDE AND MACRON`
`0x022D ȭ LATIN SMALL LETTER O WITH TILDE AND MACRON`
`0x0230 Ȱ LATIN CAPITAL LETTER O WITH DOT ABOVE AND MACRON`
`0x0231 ȱ LATIN SMALL LETTER O WITH DOT ABOVE AND MACRON`
`0x0232 Ȳ LATIN CAPITAL LETTER Y WITH MACRON`
`0x0233 ȳ LATIN SMALL LETTER Y WITH MACRON`
`0x0237 ȷ LATIN SMALL LETTER DOTLESS J`
`0x0259 ə LATIN SMALL LETTER SCHWA`
`0x02B9 ʹ MODIFIER LETTER PRIME`
`0x02BA ʺ MODIFIER LETTER DOUBLE PRIME`
`0x02BC ʼ MODIFIER LETTER APOSTROPHE`
`0x02C7 ˇ CARON`
`0x02D8 ˘ BREVE`
`0x02D9 ˙ DOT ABOVE`
`0x02DB ˛ OGONEK`
`0x02DD ˝ DOUBLE ACUTE ACCENT`
`0x0300 ̀ COMBINING GRAVE ACCENT`
`0x0301 ́ COMBINING ACUTE ACCENT`
`0x0302 ̂ COMBINING CIRCUMFLEX ACCENT`
`0x0303 ̃ COMBINING TILDE`
`0x0304 ̄ COMBINING MACRON`
`0x0306 ̆ COMBINING BREVE`
`0x0307 ̇ COMBINING DOT ABOVE`
`0x0308 ̈ COMBINING DIAERESIS`
`0x0309 ̉ COMBINING HOOK ABOVE`
`0x030A ̊ COMBINING RING ABOVE`
`0x030B ̋ COMBINING DOUBLE ACUTE ACCENT`
`0x030C ̌ COMBINING CARON`
`0x030F ̏ COMBINING DOUBLE GRAVE ACCENT`
`0x0311 ̑ COMBINING INVERTED BREVE`
`0x0312 ̒ COMBINING TURNED COMMA ABOVE`
`0x031B ̛ COMBINING HORN`
`0x0323 ̣ COMBINING DOT BELOW`
`0x0324 ̤ COMBINING DIAERESIS BELOW`
`0x0326 ̦ COMBINING COMMA BELOW`
`0x0327 ̧ COMBINING CEDILLA`
`0x0328 ̨ COMBINING OGONEK`
`0x032E ̮ COMBINING BREVE BELOW`
`0x0331 ̱ COMBINING MACRON BELOW`
`0x0335 ̵ COMBINING SHORT STROKE OVERLAY`
`0x1E80 Ẁ LATIN CAPITAL LETTER W WITH GRAVE`
`0x1E81 ẁ LATIN SMALL LETTER W WITH GRAVE`
`0x1E82 Ẃ LATIN CAPITAL LETTER W WITH ACUTE`
`0x1E83 ẃ LATIN SMALL LETTER W WITH ACUTE`
`0x1E84 Ẅ LATIN CAPITAL LETTER W WITH DIAERESIS`
`0x1E85 ẅ LATIN SMALL LETTER W WITH DIAERESIS`
`0x1E9E ẞ LATIN CAPITAL LETTER SHARP S`
`0x1EA0 Ạ LATIN CAPITAL LETTER A WITH DOT BELOW`
`0x1EA1 ạ LATIN SMALL LETTER A WITH DOT BELOW`
`0x1EA2 Ả LATIN CAPITAL LETTER A WITH HOOK ABOVE`
`0x1EA3 ả LATIN SMALL LETTER A WITH HOOK ABOVE`
`0x1EA4 Ấ LATIN CAPITAL LETTER A WITH CIRCUMFLEX AND ACUTE`
`0x1EA5 ấ LATIN SMALL LETTER A WITH CIRCUMFLEX AND ACUTE`
`0x1EA6 Ầ LATIN CAPITAL LETTER A WITH CIRCUMFLEX AND GRAVE`
`0x1EA7 ầ LATIN SMALL LETTER A WITH CIRCUMFLEX AND GRAVE`
`0x1EA8 Ẩ LATIN CAPITAL LETTER A WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1EA9 ẩ LATIN SMALL LETTER A WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1EAA Ẫ LATIN CAPITAL LETTER A WITH CIRCUMFLEX AND TILDE`
`0x1EAB ẫ LATIN SMALL LETTER A WITH CIRCUMFLEX AND TILDE`
`0x1EAC Ậ LATIN CAPITAL LETTER A WITH CIRCUMFLEX AND DOT BELOW`
`0x1EAD ậ LATIN SMALL LETTER A WITH CIRCUMFLEX AND DOT BELOW`
`0x1EAE Ắ LATIN CAPITAL LETTER A WITH BREVE AND ACUTE`
`0x1EAF ắ LATIN SMALL LETTER A WITH BREVE AND ACUTE`
`0x1EB0 Ằ LATIN CAPITAL LETTER A WITH BREVE AND GRAVE`
`0x1EB1 ằ LATIN SMALL LETTER A WITH BREVE AND GRAVE`
`0x1EB2 Ẳ LATIN CAPITAL LETTER A WITH BREVE AND HOOK ABOVE`
`0x1EB3 ẳ LATIN SMALL LETTER A WITH BREVE AND HOOK ABOVE`
`0x1EB4 Ẵ LATIN CAPITAL LETTER A WITH BREVE AND TILDE`
`0x1EB5 ẵ LATIN SMALL LETTER A WITH BREVE AND TILDE`
`0x1EB6 Ặ LATIN CAPITAL LETTER A WITH BREVE AND DOT BELOW`
`0x1EB7 ặ LATIN SMALL LETTER A WITH BREVE AND DOT BELOW`
`0x1EB8 Ẹ LATIN CAPITAL LETTER E WITH DOT BELOW`
`0x1EB9 ẹ LATIN SMALL LETTER E WITH DOT BELOW`
`0x1EBA Ẻ LATIN CAPITAL LETTER E WITH HOOK ABOVE`
`0x1EBB ẻ LATIN SMALL LETTER E WITH HOOK ABOVE`
`0x1EBC Ẽ LATIN CAPITAL LETTER E WITH TILDE`
`0x1EBD ẽ LATIN SMALL LETTER E WITH TILDE`
`0x1EBE Ế LATIN CAPITAL LETTER E WITH CIRCUMFLEX AND ACUTE`
`0x1EBF ế LATIN SMALL LETTER E WITH CIRCUMFLEX AND ACUTE`
`0x1EC0 Ề LATIN CAPITAL LETTER E WITH CIRCUMFLEX AND GRAVE`
`0x1EC1 ề LATIN SMALL LETTER E WITH CIRCUMFLEX AND GRAVE`
`0x1EC2 Ể LATIN CAPITAL LETTER E WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1EC3 ể LATIN SMALL LETTER E WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1EC4 Ễ LATIN CAPITAL LETTER E WITH CIRCUMFLEX AND TILDE`
`0x1EC5 ễ LATIN SMALL LETTER E WITH CIRCUMFLEX AND TILDE`
`0x1EC6 Ệ LATIN CAPITAL LETTER E WITH CIRCUMFLEX AND DOT BELOW`
`0x1EC7 ệ LATIN SMALL LETTER E WITH CIRCUMFLEX AND DOT BELOW`
`0x1EC8 Ỉ LATIN CAPITAL LETTER I WITH HOOK ABOVE`
`0x1EC9 ỉ LATIN SMALL LETTER I WITH HOOK ABOVE`
`0x1ECA Ị LATIN CAPITAL LETTER I WITH DOT BELOW`
`0x1ECB ị LATIN SMALL LETTER I WITH DOT BELOW`
`0x1ECC Ọ LATIN CAPITAL LETTER O WITH DOT BELOW`
`0x1ECD ọ LATIN SMALL LETTER O WITH DOT BELOW`
`0x1ECE Ỏ LATIN CAPITAL LETTER O WITH HOOK ABOVE`
`0x1ECF ỏ LATIN SMALL LETTER O WITH HOOK ABOVE`
`0x1ED0 Ố LATIN CAPITAL LETTER O WITH CIRCUMFLEX AND ACUTE`
`0x1ED1 ố LATIN SMALL LETTER O WITH CIRCUMFLEX AND ACUTE`
`0x1ED2 Ồ LATIN CAPITAL LETTER O WITH CIRCUMFLEX AND GRAVE`
`0x1ED3 ồ LATIN SMALL LETTER O WITH CIRCUMFLEX AND GRAVE`
`0x1ED4 Ổ LATIN CAPITAL LETTER O WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1ED5 ổ LATIN SMALL LETTER O WITH CIRCUMFLEX AND HOOK ABOVE`
`0x1ED6 Ỗ LATIN CAPITAL LETTER O WITH CIRCUMFLEX AND TILDE`
`0x1ED7 ỗ LATIN SMALL LETTER O WITH CIRCUMFLEX AND TILDE`
`0x1ED8 Ộ LATIN CAPITAL LETTER O WITH CIRCUMFLEX AND DOT BELOW`
`0x1ED9 ộ LATIN SMALL LETTER O WITH CIRCUMFLEX AND DOT BELOW`
`0x1EDA Ớ LATIN CAPITAL LETTER O WITH HORN AND ACUTE`
`0x1EDB ớ LATIN SMALL LETTER O WITH HORN AND ACUTE`
`0x1EDC Ờ LATIN CAPITAL LETTER O WITH HORN AND GRAVE`
`0x1EDD ờ LATIN SMALL LETTER O WITH HORN AND GRAVE`
`0x1EDE Ở LATIN CAPITAL LETTER O WITH HORN AND HOOK ABOVE`
`0x1EDF ở LATIN SMALL LETTER O WITH HORN AND HOOK ABOVE`
`0x1EE0 Ỡ LATIN CAPITAL LETTER O WITH HORN AND TILDE`
`0x1EE1 ỡ LATIN SMALL LETTER O WITH HORN AND TILDE`
`0x1EE2 Ợ LATIN CAPITAL LETTER O WITH HORN AND DOT BELOW`
`0x1EE3 ợ LATIN SMALL LETTER O WITH HORN AND DOT BELOW`
`0x1EE4 Ụ LATIN CAPITAL LETTER U WITH DOT BELOW`
`0x1EE5 ụ LATIN SMALL LETTER U WITH DOT BELOW`
`0x1EE6 Ủ LATIN CAPITAL LETTER U WITH HOOK ABOVE`
`0x1EE7 ủ LATIN SMALL LETTER U WITH HOOK ABOVE`
`0x1EE8 Ứ LATIN CAPITAL LETTER U WITH HORN AND ACUTE`
`0x1EE9 ứ LATIN SMALL LETTER U WITH HORN AND ACUTE`
`0x1EEA Ừ LATIN CAPITAL LETTER U WITH HORN AND GRAVE`
`0x1EEB ừ LATIN SMALL LETTER U WITH HORN AND GRAVE`
`0x1EEC Ử LATIN CAPITAL LETTER U WITH HORN AND HOOK ABOVE`
`0x1EED ử LATIN SMALL LETTER U WITH HORN AND HOOK ABOVE`
`0x1EEE Ữ LATIN CAPITAL LETTER U WITH HORN AND TILDE`
`0x1EEF ữ LATIN SMALL LETTER U WITH HORN AND TILDE`
`0x1EF0 Ự LATIN CAPITAL LETTER U WITH HORN AND DOT BELOW`
`0x1EF1 ự LATIN SMALL LETTER U WITH HORN AND DOT BELOW`
`0x1EF2 Ỳ LATIN CAPITAL LETTER Y WITH GRAVE`
`0x1EF3 ỳ LATIN SMALL LETTER Y WITH GRAVE`
`0x1EF4 Ỵ LATIN CAPITAL LETTER Y WITH DOT BELOW`
`0x1EF5 ỵ LATIN SMALL LETTER Y WITH DOT BELOW`
`0x1EF6 Ỷ LATIN CAPITAL LETTER Y WITH HOOK ABOVE`
`0x1EF7 ỷ LATIN SMALL LETTER Y WITH HOOK ABOVE`
`0x1EF8 Ỹ LATIN CAPITAL LETTER Y WITH TILDE`
`0x1EF9 ỹ LATIN SMALL LETTER Y WITH TILDE`
`0x2010 ‐ HYPHEN`
`0x2020 † DAGGER`
`0x2020 † DAGGER`
`0x2021 ‡ DOUBLE DAGGER`
`0x2021 ‡ DOUBLE DAGGER`
`0x2030 ‰ PER MILLE SIGN`
`0x2032 ′ PRIME`
`0x2033 ″ DOUBLE PRIME`
`0x2052 ⁒ COMMERCIAL MINUS SIGN`
`0x20A1 ₡ COLON SIGN`
`0x20A3 ₣ FRENCH FRANC SIGN`
`0x20A4 ₤ LIRA SIGN`
`0x20A6 ₦ NAIRA SIGN`
`0x20A7 ₧ PESETA SIGN`
`0x20A9 ₩ WON SIGN`
`0x20AB ₫ DONG SIGN`
`0x20AD ₭ KIP SIGN`
`0x20B1 ₱ PESO SIGN`
`0x20B2 ₲ GUARANI SIGN`
`0x20B5 ₵ CEDI SIGN`
`0x20B9 ₹ INDIAN RUPEE SIGN`
`0x20BA ₺ TURKISH LIRA SIGN`
`0x20BC ₼ MANAT SIGN`
`0x20BD ₽ RUBLE SIGN`
`0x2116 № NUMERO SIGN`
`0x2122 ™ TRADE MARK SIGN`
`0x2219 ∙ BULLET OPERATOR`
`0x2248 ≈ ALMOST EQUAL TO`
`0x2260 ≠ NOT EQUAL TO`
`0x2264 ≤ LESS-THAN OR EQUAL TO`
`0x2265 ≥ GREATER-THAN OR EQUAL TO`
`0x27E8 ⟨ MATHEMATICAL LEFT ANGLE BRACKET`
`0x27E9 ⟩ MATHEMATICAL RIGHT ANGLE BRACKET`
`0xFB01 ﬁ LATIN SMALL LIGATURE FI`
`0xFB02 ﬂ LATIN SMALL LIGATURE FL`
`f_f`
`f_f_i`
`f_f_l`
`circumflexcomb_acutecomb`
`circumflexcomb_gravecomb`
`circumflexcomb_hookabovecomb`
`circumflexcomb_tildecomb`
`brevecomb_acutecomb`
`brevecomb_gravecomb`
`brevecomb_hookabovecomb`
`brevecomb_tildecomb`
`idotaccent`
`caroncomb.alt`
`zero.lf`
`one.lf`
`two.lf`
`three.lf`
`four.lf`
`five.lf`
`six.lf`
`seven.lf`
`eight.lf`
`nine.lf`
`periodcentered.loclCAT`

个久以前在大征很我星是海的系辰远途遥

**Comment by davelab6** [comment]
Is this done for Roman and Italic?

**Comment by dberlow** [comment]
The glyphs are all drawn for Roman and Italic.

I am making measurements from them for the new sheet.

**Comment by dberlow** [comment]
Should add, Chinese is not there yet

**Comment by sberlow** [comment]
@davelab6 should we put the Chinese on hold for this product and put in a future release?

**Comment by dberlow** [comment]
Well whatever you guys decide is fine with me.  
I’m measuring, documenting, and ironing out the axes list, glyph repertoire and feature list into perfect varmony with current varchitecture, for the final process and checklist.

**Comment by davelab6** [comment]
I see the goal of the current round of development as being to make the
highest-quality production-ready VF serif face using David's axes system
possible, so I'm fine to drop the chinese experimental stuff out, it is
still available in the final AmstelvarAlpha release for those curious

**Comment by davelab6** [comment]
Before your measurements I think we need to go through a round of
presentation and approval of the default master designs for both.

**Comment by dberlow** [comment]
Too late. You want me to unmeasure them? You want your money back for good measure? You want to compare the before and after measure for measure, I doubt it will change. Most of this is staying the same as the Alpha dog.

**Comment by davelab6** [comment]
If you are well into the measurements phase then lets do the review and any
design changes can be remeasured?

To confirm, the measuring needs to be done before proceeding with pole
master drawing, and that hasn't started yet?

**Comment by dberlow** [comment]
Pole mastering has not started.

**Comment by davelab6** [comment]
Then I think we are ahead of schedule but still on track :D

**Comment by sberlow** [comment]
These glyphs are all drawn.
closing
